### DISCUSSION TITLE
vNext

### DISCUSSION THREAD

**Discussion by danjagnow** [danjagnow/ArgentPonyWarcraftClient#176] [Ideas]
Now that [milestone 5.0](https://github.com/blizzard-net/warcraft/milestone/8) is complete and the new APIs for the Shadowlands expansion are supported, what comes next?  Possibilities:

- [Enable nullable reference types](https://github.com/blizzard-net/warcraft/issues/170)
- Continue to improve the robustness of the tests that @willwolfram18 has started.
- Consider the tradeoffs of updating the [System.Text.Json](https://www.nuget.org/packages/System.Text.Json/) dependency from 4.7.2 to 5.0.0.
- Would it help to have a sample project or two to demonstrate API usage?
- What about more comprehensive help with a tool like [Docusaurus](https://v2.docusaurus.io/) (broadly adopted, easy to get started) or [DocFX](https://dotnet.github.io/docfx/) (more .NET-centric, with better support for generating docs from XML documentation).
- Other ideas?

**Comment by willwolfram18** [comment]
I'm going to try working on adding more to the set of tests that we have for the different APIs, specifically with the Integration test project. I figure the integration tests will give us the best visibility into when breaking changes, or even just changes we're not aware of, are introduced by Blizzard which will allow us to at least react quicker :+1:
